R/functions_io.R

```diff
@@ -338,8 +345,9 @@ ReadSparseMatrix = function(path, project="SeuratProject", row_name_column=2, co
 #' @param feature_type_to_assay_name How should the feature types for the different assays be named? Default is: "RNA" = "Gene expression","ADT" = "Antibody Capture" ,"HTO" = "Antibody Capture","Crispr" = "CRISPR Guide Capture", "ERCC" = "ERCC" and "Custom" = "Custom". Also sets the order in which the feature types are written.
 #' @param include_cell_metadata_cols Vector with names of cell metadata columns to be written to the metadata.tsv.gz. Can be NULL in which case no file is created.
 #' @param include_feature_metadata_cols Vector with names of feature metadata columns to be written. Can be NULL in which case only the first three columns (feature id, feature symbol and feature type) are written.
+#' @param metadata_prefix Prefix for cell metadata column names. If NULL, no prefix. Separator must be included.
```

Comment:
I don't get this one. What do you mean by "Separator must be included."?

If NULL, no prefix is added.

Reply:
I think we take care of this when we do the output (so after this PR)

Reply:
Can you for now rewrite the comment, I don't find it very clear. 
Prefix for cell metadata column names. If NULL, no prefix is prepended. The prefix should end with a separator character that separates the prefix from the original column name.

Reply:
okay I will do this